docs/aws/audit/cloudwatchmonitoring/rules/iam_policy_changes_alarm.mdx

```diff
@@ -24,101 +24,71 @@ SOC2, HIPAA, ISO27001, AWSWAF, HITRUST, CISAWS, CBP, NISTCSF, PCIDSS
 ```
-aws iam get-group-policy --group-name <group-name> --policy-name <policy-name>
-```
-
-Replace `<group-name>` and `<policy-name>` with the actual names of the group and policy that were modified.
 
-4. If the policy changes were unauthorized or unintended, you should revoke the user or role's permissions to modify IAM policies. You can do this by removing the "iam:PutGroupPolicy" permission from the user or role's IAM policy.
+In this example:
+- `MyNamespace` is the namespace for the metric.
+- `MyMetric` is the name of the metric.
+- `1` is the value of the metric.
+- `$(date -u +'%Y-%m-%dT%H:%M:%S')` is the timestamp in ISO 8601 format.
+- `us-east-1` is the AWS region.
 
-5. To remove the "iam:PutGroupPolicy" permission from an IAM policy, you can use the following command:
+Replace these values with your actual metric data. Also, ensure that the AWS CLI is configured with the correct AWS region where your CloudWatch metrics are located.
 
-```
-aws iam update-assume-role-policy --role-name <role-name> --policy-document file://policy.json
-```
-
-Replace `<role-name>` with the actual name of the role that needs to be updated, and `policy.json` with a file containing the updated policy document that removes the "iam:PutGroupPolicy" permission.
-
-6. After making the necessary changes, monitor the CloudTrail logs to ensure that no further unauthorized or unintended changes are made to IAM policies.
+Remember that you can't directly create metrics in CloudWatch through the AWS CLI. You can only publish data points to existing custom metrics or predefined AWS service metrics. If you're working with custom metrics, make sure you have set up the metric namespace and dimensions beforehand.
 
 #### Using Python
 
-The IAM Policy Changes Alarm is triggered when a change is made to an AWS Identity and Access Management (IAM) policy. This can be a security risk if the policy is changed in a way that grants excessive permissions to users or roles. Here are the steps to remediate this issue using Python:
-
-1. Identify the root cause of the policy change. This could be a user or a role that made the change.
-
-2. Review the IAM policy change and determine if it was authorized or unauthorized.
-
-3. If the policy change was unauthorized, revoke the IAM permissions of the user or role that made the change.
-
-4. If the policy change was authorized, review the change to ensure that it does not grant excessive permissions.
-
-5. If the change does grant excessive permissions, modify the policy to remove the excessive permissions.
-
-6. Monitor the IAM policy changes using AWS CloudTrail to ensure that there are no further unauthorized changes.
-
-Here's an example Python script to revoke IAM permissions for a user:
+Here's how you can check for the existence of the metric `IAMPolicyChangesEventCount` using Python with the boto3 library:
```

Comment:
same with python steps as well. it does not have remediation steps. It only has detection steps.